Prototope/Geometry.swift

```diff
@@ -80,6 +80,11 @@ public func *(scalar: Double, a: Point) -> Point {
     return a * scalar
 }
 
+
+public func*(a: Point, b: Point) -> Point {
```

Comment:
Probably worth documenting here that this is element-wise multiplication, not an outer product or something: there are a number of valid "multiplications" for points/vectors.